src/main/java/frc/robot/commands/Teleop/FloorPickup.java

```diff
@@ -28,12 +32,16 @@ public void initialize() {
 
   // Called every time the scheduler runs while the command is scheduled.
   @Override
-  public void execute() {}
+  public void execute() {
+    if (intakePivot.getPosition() > Constants.IntakeConstants.DROPPED) {
+      intakePivot.setHingePower(0.0);
+    }
+  }
 
   // Called once the command ends or is interrupted.
   @Override
   public void end(boolean interrupted) {
-    intakePivot.setHingePower(0.0);
+    intake.setRollerPower(0.0);
```

Comment:
You still need to stop the pivot here in the case that the button is released/command ends early

Reply:
done